src/ansi-c/c_preprocess.cpp

```diff
@@ -951,37 +707,6 @@ bool c_preprocess_arm(
 
   command="armcc -E -D__CPROVER__";
 
-//  command+=" -D__sizeof_int="+std::to_string(config.ansi_c.int_width/8);
-//  command+=" -D__sizeof_long="+std::to_string(config.ansi_c.long_int_width/8);
-//  command+=" -D__sizeof_ptr="+std::to_string(config.ansi_c.pointer_width/8);
-  // command+=" -D__EDG_VERSION__=308";
-  // command+=" -D__EDG__";
-//  command+=" -D__CC_ARM=1";
-  // command+=" -D__ARMCC_VERSION=410000";
-//  command+=" -D__arm__";
-
-//  if(config.ansi_c.endianness==configt::ansi_ct::IS_BIG_ENDIAN)
-//    command+=" -D__BIG_ENDIAN";
```

Comment:
If armcc has the same options as GCC, then this should translate to `-mbig-endian`

Reply:
Yeah, armcc has `--bigend` (note the double `-`)